cmd/stack_package.go

```diff
@@ -230,6 +232,9 @@ func newStackPackageCmd(rootConfig *RootCommandConfig) *cobra.Command {
 				return errors.Errorf("Error reading directories: %v", err)
 			}
 
+			repoDirName := filepath.Base(filepath.Dir(stackPath))
```

Comment:
We can't count on the repo name to be in the path. A user can run this command from any path in their filesystem.